Classes/KIFUIViewTestActor.m

```diff
@@ -465,6 +466,27 @@ - (void)selectPickerViewRowWithTitle:(NSString *)title inComponent:(NSInteger)co
     }
 }
 
+#pragma mark - Date Picker Actions
+
+- (void)selectDatePickerDate:(NSDate *)date
+{
+    @autoreleasepool {
+        KIFUIObject *found = [[self _usingExpectedClass:[UIDatePicker class]] _predicateSearchWithRequiresMatch:NO mustBeTappable:NO];
```

Comment:
Can we add a predicate to ensure `datePickerMode` is the right type here and below, rather than asserting if we match on the wrong object?

Reply:
Isn't that what we are doing with `[self _usingExpectedClass:[UIPickerView class]]`